include/linux/soundwire/sdw.h

```diff
@@ -1020,6 +1021,7 @@ struct sdw_bus {
 	struct sdw_bus_params params;
 	int stream_refcount;
 	int bpt_stream_refcount;
+	struct sdw_stream_runtime *bpt_stream;
```

Comment:
it is interesting that we don't need 'normal' stream pointer for convenience, only for bpt.

Reply:
so it is not for mere convenience, but necessity ;)